src/reducers/Material.js

```diff
@@ -36,7 +36,8 @@ function materialsCloneOne(state, action) {
 }
 
 function materialsToggleIsNonPeriodicForOne(state, action) {
-    const newMaterial = state.materials[state.index].clone({hash: ""});
+    NPMsAlert.warning("Start from a cloned material before converting to 'Non-Periodic'");
+    const newMaterial = state.materials[state.index].clone({hash: "", scaledHash: ""});
```

Comment:
scaledHash also requires the derivedProperties to be created for a non-periodic structure.

Reply:
Wait - but why are we proceeding with the operation? I thought we agreed to prevent modifying existing materials (materials that already have an `_id`) and force users to create a clone first.

Reply:
I think both options got mentioned, but yes blocking the operation is probably better until we look at what is happening when a pre-existing material is changed in more detail.